### PR TITLE
Add Windows documentation for tunnelling.

### PR DESCRIPTION
**Pull request by ehiggs**
nb, this has my vsc user id. Which I'm ok with.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -27,4 +27,5 @@ Introductory topics
     Configuration
     Logging
     Connecting_to_web_UIs
+    Configuring_PuTTY_on_Windows
```
Comment (boegel) [inline docs/index.rst]: don't include it here, it should be linked from the the "Connecting to web UIs" page?

File: docs/Configuring_PuTTY_on_Windows.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+To configure PuTTY on Windows we need to do the following tasks:
+
+1. Download the required software.
+2. Add our key to Pageant.
+3. Configure PuTTY.
+
+Download the required software
+------------------------------
+
+Windows is notoriously hostile to software development so none of the technology
```
Comment (boegel) [inline docs/Configuring_PuTTY_on_Windows.rst]: haha.

no, don't include statements like this in documentation :)

Comment (JensTimmerman) [inline docs/Configuring_PuTTY_on_Windows.rst]: why not? just put a citation ;-)

Comment (ehiggs) [inline docs/Configuring_PuTTY_on_Windows.rst]: I guess people who choose to ice skate up hill will be offended?

**Comment by boegel** [comment]
Merging to see how this renders, I'll look into completing my PR with this included, thanks a bunch @ehiggs!

**Pull request merged**